### PR TITLE
feat: colors and fonts

### PR DESCRIPTION
**Pull request by chosww**

### CODE REVIEW

File: src/_includes/layout.njk [hunk 1]
```diff
@@ -8,8 +8,7 @@
     <meta property="og:type" content="website" />
     <!-- Replace with your own assets / build system -->
     <link rel="stylesheet" href="/static/css/reset.css" />
-    <link rel="stylesheet" href="/static/css/button.css" />
-    <link rel="stylesheet" href="/static/css/quote.css" />
+    <link rel="stylesheet" href="/static/css/variables.css" />
```
Comment (greatislander) [inline src/_includes/layout.njk:11]: You will want to add `link.css` too, when ready.

File: src/static/css/variables.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+:root {
+    /* Colors */
+    --color-blue-100: hsl(234, 100%, 96%);
```
Comment (greatislander) [inline src/static/css/variables.css:3]: As per our earlier discussion, let's make these two variables like so:

```css
    --color-blue-100-hsl: 234, 100%, 96%;
    --color-blue-100: hsl(var(--color-blue-100-hsl);
```

That way we can use the HSL string variable to construct different opacity variants, like this:

```css
    --color-blue-100-50-percent: hsl(var(--color-blue-100-hsl) / 50%);
```

File: src/static/css/variables.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    --color-green-300: hsl(170, 54%, 89%);
+    --color-green-500: hsl(175, 71%, 69%);
+    --color-green-700: hsl(175, 80%, 16%);
+    --color-red-100: hsl(0, 100%, 97%);
+    --color-red-500: hsl(0, 89%, 69%);
+    --color-orange-100: hsl(40, 100%, 97%);
+    --color-orange-500: hsl(40, 100%, 49%);
+
+    /* Fonts */
+    --font-sans: Martel;
```
Comment (greatislander) [inline src/static/css/variables.css:16]: Important to add fallbacks to `serif` or `sans-serif` as appropriate, and possibly also some cross-platform system fonts (see: https://systemfontstack.com/)

**Review by greatislander — CHANGES_REQUESTED** [review]

File: src/static/css/variables.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+.flex-align--y {
+    justify-items: center;
+}
+
+/* Gap */
+
+.gap--1 {
+    gap: 1rem;
```
Comment (greatislander) [inline src/static/css/variables.css:113]: See above re: fluid scale.

**Review by greatislander — CHANGES_REQUESTED** [review]
Some changes to discuss.

File: src/static/css/variables.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    --font-serif: Mulish, Iowan Old Style, Apple Garamond, Baskerville, Times New Roman, Droid Serif, Times, Source Serif Pro, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
+}
+
+.border--full {
+    border-radius: 50%;
+}
+
+/* Width */
+
+.width--1 {
```
[the remark this answers is not in the document]
Comment (greatislander) [reply src/static/css/variables.css:37]: https://utopia.fyi

File: src/static/css/utilities.css [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+/* Gap */
+
+.gap-1 {
+    gap: var(--space-s);
+}
+
+.gap-2 {
+    gap: var(--space-m);
+}
```
Comment (greatislander) [inline src/static/css/utilities.css:26]: These should be named to match the space variables. So: `.gap-s` and `.gap-m`.

File: src/static/css/utilities.css [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+/* Height */
+
+.height-7 {
```
Comment (greatislander) [inline src/static/css/utilities.css:30]: Should be `.height-3xl`.

File: src/static/css/utilities.css [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+/* Margin */
+
+.margin-auto {
+    margin: auto;
+}
+
+/* Width */
+
+.width-7 {
```
Comment (greatislander) [inline src/static/css/utilities.css:42]: Should be `.width-3xl`.

File: src/static/css/variables.css [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+        baskerville,
+        times new roman,
+        droid serif,
+        times,
+        source serif pro,
+        apple color emoji,
+        segoe ui emoji,
+        segoe ui symbol;
+
+    /* Spacings */
```
Comment (greatislander) [inline src/static/css/variables.css:55]: Can you add the link comment from the Utopia generator for reference before all the fluid variables? Looks like this I think:

```css
/* @link https://utopia.fyi/space/calculator?c=320,18,1.2,1240,20,1.25,5,2,&s=0.75|0.5|0.25,1.5|2|3|4|6,s-l&g=s,l,xl,12 */
```

File: src/static/css/variables.css [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
```
Comment (greatislander) [inline src/static/css/variables.css:56]: Remove this empty line between the comment and the rules.

File: src/static/css/utilities.css [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+/* Height */
+
+.height-7 {
+    height: var(--space-3xl);
+}
+
+/* Margin */
+
+.margin-auto {
+    margin: auto;
```
Comment (greatislander) [inline src/static/css/utilities.css:37]: I'd prefer this to be specific to the horizontal axis, e.g.:

```css
.mx-auto {
	margin-left: auto;
	margin-right: auto;
}
```

(See: https://tailwindcss.com/docs/margin#add-horizontal-margin)

**Review by greatislander — CHANGES_REQUESTED** [review]

File: src/static/css/utilities.css [hunk 12]
```diff
@@ -16,29 +14,26 @@
...
 
-.gap-2 {
+.gap-m {
     gap: var(--space-m);
 }
 
 /* Height */
-
-.height-7 {
+.height-3x1 {
```
Comment (greatislander) [inline src/static/css/utilities.css:26]: Typo:

```suggestion
.height-3xl {
```

File: src/static/css/utilities.css [hunk 13]
```diff
@@ -16,29 +14,26 @@
...
-    margin: auto;
+.mx-auto {
+    margin-left: auto;
+    margin-right: auto;
 }
 
 /* Width */
-
-.width-7 {
+.width-3x1 {
```
Comment (greatislander) [inline src/static/css/utilities.css:37]: Typo:

```suggestion
.height-3xl {
```

File: src/static/css/variables.css (hunk 9 above)
Comment (greatislander) [reply src/static/css/variables.css:55]: Sorry, I should have clarified— I was suggesting the link to the specific configuration you used (which has query parameters on the end of it).

Also, the link should appear before the `/* Spacing */` comment as it is a header for all of the remaining blocks.

**Review by greatislander — CHANGES_REQUESTED** [review]

File: src/static/css/utilities.css (hunk 12 above)
Comment (chosww) [reply src/static/css/utilities.css:26]: I really don't know how I made this mistake :)

File: src/static/css/main.css [hunk 14]
```diff
@@ -1,2 +1,5 @@
 @import "reset.css";
 @import "link.css";
+@import "color.css";
```
Comment (greatislander) [inline src/static/css/main.css:3]: Order is important here— you'll want to start with `reset`, then `variables`, then the rest.

**Review by greatislander — APPROVED** [review]

**Pull request merged**